test/pdal_ign_macro/test_mark_points_to_use_for_digital_models_with_new_dimension.py

```diff
@@ -117,14 +116,15 @@ def test_main_with_buffer():
             tile_width=50,
             tile_coord_scale=10,
         )
-        pipeline = pdal.Pipeline()
-        pipeline |= pdal.Reader.las(las_output.name)
-        output_dimensions = pipeline.quickinfo["readers.las"]["dimensions"].split(", ")
+
+        pipeline_mtd = pdal.Reader.las(las_output.name).pipeline()
+        output_dimensions = pipeline_mtd.quickinfo["readers.las"]["dimensions"]
         assert dsm_dimension in output_dimensions
         assert dtm_dimension in output_dimensions
 
-        pipeline.execute()
-        arr = pipeline.arrays[0]
+        pipeline_out = pdal.Reader.las(las_output.name).pipeline()
```

Comment:
commentaire du style
pipeline.quickinfo done before need that we re create the pipeline